frontend/src/v5/ui/controls/formDatePicker/formBaseCalendarPicker/formBaseCalendarPicker.component.tsx

```diff
@@ -55,7 +56,11 @@ export const FormBaseCalendarPicker = ({
 					{...field}
 					{...props}
 					onOpen={() => setOpen(true)}
-					onClose={() => setOpen(false)}
+					onClose={() => {
+						// This is to signal that the date has changed (we are using onblur to save changes)
+						(props.onBlur || noop)();
```

Comment:
**This is not a change request**
When I need to invoke an optional function, I use optional chaining without resorting to `noop` functions
```ts
props.onBlur?.()
```

Reply:
**This is not a change request**
When I need to invoke an optional function, I use optional chaining without resorting to `noop` functions
```ts
props.onBlur?.()
```